build/localnet/manifests/cli-client.yaml

```diff
@@ -30,6 +30,13 @@ spec:
           image: client-image
           imagePullPolicy: IfNotPresent
           command: ["sleep", "infinity"]
+          resources:
+            requests:
+              memory: "64Mi"
+              cpu: "250m"
+            limits:
+              memory: "512Mi"
```

Comment:
This makes the `client` process die quick instead of crashing control plane when memory is full.

Reply:
Did you check if this works with docker-compose client?

Reply:
> Did you check if this works with docker-compose client?

Yep, it worked for me!